examples/server/README.md

```diff
@@ -65,6 +65,14 @@ server.exe -m models\7B\ggml-model.gguf -c 2048
 The above command will start a server that by default listens on `127.0.0.1:8080`.
 You can consume the endpoints with Postman or NodeJS with axios library. You can visit the web front end at the same url.
 
+### Docker:
+```bash
+docker run -p 8080:8080 -v /path/to/models:/models ggerganov/llama.cpp:server -m models/7B/ggml-model.gguf -c 512 --host 0.0.0.0 --port 8080
+
+# or, with CUDA:
+docker run -p 8080:8080 -v /path/to/models:/models --gpus all ggerganov/llama.cpp:server -m models/7B/ggml-model.gguf -c 512 --host 0.0.0.0 --port 8080 --n-gpu-layers 1
```

Comment:
Maybe bump the `ngl` to 99 in the instructions to avoid under-utilizing VRAM:

```suggestion
docker run -p 8080:8080 -v /path/to/models:/models --gpus all ggerganov/llama.cpp:server -m models/7B/ggml-model.gguf -c 512 --host 0.0.0.0 --port 8080 --n-gpu-layers 99
```

Reply:
I always use `-1` personally, I just used `1` to be consistent with elsewhere in the README ([here](https://github.com/ggerganov/llama.cpp/blob/6db2b41a76ee78d5efdd5c3cddd5d7ad3f646855/README.md?plain=1#L995)), but happy to update it :)

Reply:
I just updated it though :)